problem-14.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+// Prints a diamond made of asterisks(*) of a specified size
+
+int main()
+{
+    int size, i, j;
+
+    printf("---------DIAMOND FROM ASTERISKS---------\n");
+
+    printf("Enter the size of diamond: ");
+    scanf("%d", &size);
+
+    int gap = size - 1;

```

Comment:
How about we add some comment here that explains what `gap` is?

Reply:
@M-Ahmed-Khan we should take care of grammar in comments i.e. first letter capital and full stop at end.